### PR TITLE
added registration form

### PR DESCRIPTION
**Pull request by StehanO**
added registration form

### CODE REVIEW

File: src/components/RegistrationForm/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      />
+      <Field
+        name="password"
+        component={TextField}
+        type="password"
+        label="Password"
+        className={styles.field}
+      />
+      <Field
+        name="confirm-password"
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/index.js]: Please change to` confirm-password` to `confirmPassword`. 
Might not be as pretty but more for naming standards.

File: src/components/RegistrationForm/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        component={TextField}
+        type="password"
+        label="Confirm Password"
+        className={styles.field}
+      />
+      <Field
+        name="firstname"
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/index.js]: `firstname` -> `firstName` as required by API

File: src/components/RegistrationForm/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        component={TextField}
+        type="text"
+        label="First Name"
+        className={styles.field}
+      />
+      <Field
+        name="lastname"
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/index.js]: `lastname` -> `lastName` as required by API

File: src/components/RegistrationForm/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        component={TextField}
+        type="text"
+        label="Last Name"
+        className={styles.field}
+      />
+      {error && <div>{error.toString()}</div>}
+    </div>
+    <Button className={styles.button} type="submit" disabled={submitting}>
+      Continue
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/index.js]: Don't know if "Continue" makes sense? Open to discussion

File: src/components/RegistrationForm/index.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Link from 'react-router-dom/Link'
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/index.js]: Unused import. Please double check eslint

File: src/components/RegistrationForm/styles.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+.fields {
+  width: 100%;
+}
+
+.field, .button {
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.link {
```
Comment (fjlaubscher) [inline src/components/RegistrationForm/styles.scss]: Doesn't seem like this style is being used on the form. safe to delete it.

File: src/components/RegistrationForm/index.js (hunk 4 above)
Comment (StehanO) [reply src/components/RegistrationForm/index.js]: Can change it to "Register"

Comment (fjlaubscher) [reply src/components/RegistrationForm/index.js]: 🤙

**Pull request merged**